deploy/agencies.json

```
@@ -38,6 +38,11 @@
     "analyticsReportIds": "470089273",
     "agencyName": "army",
     "awsBucketPath": "data/army"
+  },
+  {
+    "analyticsReportIds": "47783782",
```

```suggestion
    "analyticsReportIds": "477837824",
```
Double check me but I believe you left off a 4.